contrib/experimental/great_expectations_experimental/expectations/expect_column_to_have_no_months_missing.py

```diff
@@ -0,0 +1,191 @@
+                    months_ago[2],
+                ],
+            },
+            "suppress_test_for": ["mssql"],
+            "tests": [
+                {
+                    "title": "negative_missing_two_months",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {"column": "column_two_months_missing", "threshold": 1},
+                    "out": {"success": False},
+                },
+                {
+                    "title": "positive_missing_two_months",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {"column": "column_two_months_missing", "threshold": 4},
+                    "out": {"success": True},
+                },
+                {
+                    "title": "negative_missing_one_month",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {"column": "column_one_month_missing", "threshold": 0},
+                    "out": {"success": False},
+                },
+                {
+                    "title": "positive_none_missing",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {"column": "column_none_missing", "threshold": 2},
+                    "out": {"success": True},
+                },
+            ],
+        }
+    ]
+
+    # Setting necessary computation metric dependencies and defining kwargs, as well as assigning kwargs default values
+    metric_dependencies = ("column.distinct_months",)
+    success_keys = ("threshold",)
+
+    # Default values
+    default_kwarg_values = {}
+
+    library_metadata = {
+        "maturity": "experimental",
+        "contributors": [
+            "@hadasm",
+        ],
+        "tags": ["date-column"],
+    }
+
+    def validate_configuration(
+        self, configuration: Optional[ExpectationConfiguration]
+    ) -> None:
+        # Setting up a configuration
+        super().validate_configuration(configuration)
+
+    def _validate(
+        self,
+        configuration: ExpectationConfiguration,
+        metrics: Dict,
+        runtime_configuration: dict = None,
+        execution_engine: ExecutionEngine = None,
+    ):
+        from datetime import datetime
+
+        dist_months_as_str = metrics["column.distinct_months"]
+        distinct_months_sorted = sorted(
+            [
+                datetime.strptime(month_str, MONTH_FORMAT)
+                for month_str in dist_months_as_str
+            ]
+        )
+        min_month, max_month = distinct_months_sorted[0], distinct_months_sorted[-1]
+        months_diff = relativedelta(max_month, min_month).months
+        month_set = {
+            min_month + relativedelta(months=n_month) for n_month in range(months_diff)
+        }
+        missing_months = sorted(month_set - set(distinct_months_sorted))
```

Comment:
Nit (non-blocking): Only the length of missing_months is used after this, thus the sorted is not required here.

Reply:
Thanks for the great suggestions!